test/jest-config.js

```diff
@@ -7,6 +7,12 @@ module.exports = {
   preset: "ts-jest",
   testEnvironment: "jsdom",
   rootDir: ROOT,
+  globals: {
+    "ts-jest": {
+      tsconfig: path.resolve(ROOT, "./test/tsconfig.json"),
+      isolatedModules: true, // Disables type-checking
+    },
+  },
```

Comment:
This prevents type checking while running tests in watch mode. Since we run type checks in CI, and the IDE shows type errors, it's ok for us to disable this.

Reply:
Yes, we have `pnpm run types:check` which also runs as part of `pnpm run check`, in addition to IDE support, that should be sufficient.